src/RadioGroup.jsx

```diff
@@ -1,101 +1,66 @@
-import chainedFunction from 'chained-function';
-import React, { cloneElement, PureComponent } from 'react';
+import React from 'react';
 import PropTypes from 'prop-types';
-import RadioButton from './RadioButton';
+import { RadioGroupContext } from './context';
 
-const getComponentType = (Component) => (Component ? (<Component />).type : undefined);
-
-class RadioGroup extends PureComponent {
+class RadioGroup extends React.Component {
     static propTypes = {
         disabled: PropTypes.bool,
         name: PropTypes.string,
         value: PropTypes.any,
         defaultValue: PropTypes.any,
```

Comment:
感覺defaultValue可以拿掉。

Reply:
The defaultValue here stands for providing the ability to render an uncontrolled component. Since the defaultValue is explicitly mentioned in the documentation, I'd suggest keep it in the propTypes.